composer.json

```diff
@@ -34,7 +34,8 @@
         "symfony/yaml": "^5.0",
         "symfony/validator": "^5.0",
         "phpstan/phpstan-symfony": "^1.0",
-        "assoconnect/php-quality-config": "^1.2"
+        "assoconnect/php-quality-config": "^1.2",
+        "symfony/security-core": "^5.0"
```

Comment:
tu parles du composant Router dans ton message Slack, rien à rajouter le concernant ici ?

Reply:
We don't need it as the interface `RequestContextAwareLogFactoryInterface` only requires the log factory to be able to set the `RequestContext` to the log whatever the way to do it! So that's one less dependency 😎